content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -35,7 +35,7 @@ The **flowcontrol.apiserver.k8s.io/v1beta2** API version of FlowSchema and Prior
 
 ### v1.27
 
-The **v1.27** release will stop serving the following deprecated API versions:
+The **v1.27** release has stopped serving the following deprecated API versions post release:
```

Comment:
I'd suggest matching the language used for other versions below:


```suggestion
The **v1.27** release stopped serving the following deprecated API versions:
```

Reply:
yeah thanks, i missed out on that part completely : )